### ISSUE TITLE
sitewide, components [SCREENREADER]: Search for dropdown text is read aloud each time a letter is entered into Search city... input

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#529]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
!  Team affected: VSA Facilities; Project: Facility Locator
```

## Issue
The Search For field announces "All Facilities" repeatedly when I'm entering text into the Search city, state, or postal code field. I hadn't noticed it before, because I was typing quickly. By typing one letter at a time, slowly, I am able to verify this bug. This was flagged as a SC 4.1.1 violation. Screenshot attached.

## Audit Finding
* Note 8, Defect 2 of 3

## Acceptance Criteria
* As a screen reader user, I do not want to hear the text "All Facilities" read out each time I press a key in the Search city, state, or postal code field. I should only hear the character I entered when I am typing slowly, one key at a time.

## Environment
* Windows 10
* IE11 + JAWS
* Chrome + JAWS
* Chrome + NVDA
* Firefox + NVDA
* https://staging.va.gov/find-locations/ (Searched for "Austin, TX" and "Salt Lake City"

## Possible Fixes (optional)
There will be two fixes for this item. The first will be to isolate the Search for field. I believe this is built on the open-source [Downshift](https://github.com/downshift-js/downshift) component. During basic testing, it seems the Search city, state, or postal code `onChange` event is causing the Search for component to re-render or update props, thus announcing it on every change.

The second item is to clear the `title` attribute from the Search city, state, or postal code input. It has a properly attributed label, so the title is causing additional information to be read on some screen readers.

```html
<input
  id="street-city-state-zip"
  name="street-city-state-zip"
  type="text"
  value=""
>
```

## WCAG or Vendor Guidance (optional)
* [Parsing: Understanding SC 4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

## Screenshots or Trace Logs
![Screen Shot 2019-07-10 at 6.45.27 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/6e0d76e0-4a9d-4d54-ae43-899739d4fc5b)

**Comment by mmiddaugh** [comment]
@brangi to investigate before grooming

**Comment by mmiddaugh** [comment]
Michelle will schedule time with Jennifer to walk through this issue and understand ideal user experience so we can determine tasks to resolve.

**Comment by davidconlon** [comment]
Jennifer to clarify success criteria.

1. Land on page
2. Skip to content
3. Reads H1 and intro text
4. Search by "city state or zip code"
5. Reads out each letter user types
6. Tabs over to select drop down

**Comment by mmiddaugh** [comment]
May need to be handled more globally

**Comment by jenstrickland** [comment]
This appears to be a bug in the form input field validation, and a similar issue was found in https://github.com/department-of-veterans-affairs/va.gov-team/issues/5875

**Comment by alexpappasoddball** [comment]
@brandonrapp Throwing your way for prioritization

**Comment by mmiddaugh** [comment]
Has this been resolved? [#5323](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5323) (a related issue) was closed in February. @jenstrickland

**Comment by alexpappasoddball** [comment]
@meganhkelley I believe this is in your backlog, passed on by Brandon who is no longer on the contract!

**Comment by meganhkelley** [comment]
@caw310 passing this to VSP Design System team; not sure if this bug is still an issue.

**Comment by mmiddaugh** [comment]
@joshkimux are you able to confirm/validate whether or not this is still an issue?

**Comment by joshkimux** [comment]
@mmiddaugh this is no longer an issue. Validated on Firefox/NVDA and Safari/VoiceOver 👍

**Comment by mmiddaugh** [comment]
@meganhkelley can we close this issue?

**Comment by meganhkelley** [comment]
If it's no longer an issue I'd say so! cc @1Copenut since you opened the initial ticket
